service/storage_proxy.cc

```diff
@@ -2130,8 +2130,8 @@ paxos_response_handler::begin_and_repair_paxos(client_state& cs, unsigned& conte
                 co_await std::move(f);
             } catch(...) {
                 paxos::paxos_state::logger.debug("CAS[{}] Failure during commit repair {}", _id, std::current_exception());
-                continue;
             }
+            continue;
```

Comment:
But why we need to retry the entire loop in this case? Isn't it sufficient to just `mutate_internal(consistency_level::QUORUM)`?

I think we need way more comments in this code. Probably I should just inline my doc from [this PR](https://github.com/scylladb/scylladb/pull/23158#discussion_r2114375306)  here.

Reply:
> But why we need to retry the entire loop in this case? Isn't it sufficient to just `mutate_internal(consistency_level::QUORUM)`?
> 

QUORUM may write to more replicas that needed.